src/program.js

```diff
@@ -246,6 +246,12 @@ Example: $0 --help run.
       describe: 'Show verbose output',
       type: 'boolean',
     },
+    'ignore-files': {
+      alias: 'i',
+      describe: 'Files to ignore.',
```

Comment:
It can be relative path. [`FileFilter` resolves it with `sourceDir`](https://github.com/eight04/web-ext/blob/cae516e433e66ce6489fb9a1a65f8603f887daea/src/util/file-filter.js#L56).